packaging/debian/storagenode.links

```diff
@@ -0,0 +1,2 @@
+/var/lib/storagenode/storagenode-updater /usr/bin
```

Comment:
You probably want this line in a file called `debian/storagenode-updater.links` instead, so that this link is owned by the `storagenode-updater` package instead of by the `storagenode` package.

Reply:
why do you mean by pkg here? a separate deb?

Reply:
Yeah. The packaging here is creating two separate debs, one for `storagenode` and one for `storagenode-updater`. See `debian/control`. If we only want one deb, we should remove one of the `Package:` stanzas in `debian/control`.

Reply:
but we have only one pkg in debian control

Reply:
Oh. My mistake. There is only one package. Maybe I'm thinking of an earlier version? No, it's fine for there to be only one package. I guess I was just confused.